great-idea-website/css/index.css

```diff
@@ -63,4 +63,92 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+.container {
+	width: 878px;
+	margin: auto;
+}
+/* header */
+	header nav {
+		display: inline-block;
+		margin-top: 35px;
+		margin-left: 40px;
+	}
+
+	header nav a {
+		margin: 0 20px;
+		color: gray;
+		text-decoration: none;
```

Comment:
Your navigation anchor elements and your image is closed to being centered with each other. You could use `vertical-align` to fine tune that centering (_though moving forward, Flexbox is a much better idea_)